lib/mayaUsd/CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ target_link_libraries(${PROJECT_NAME}
         $<$<BOOL:${UFE_FOUND}>:${UFE_LIBRARY}>
         ${MAYA_LIBRARIES}
         mayaUsdUtils
+        mayaUsd_Schemas
```

Comment:
Don't think we should be adding this dependence.

Reply:
It was necessary to access the schema attribute.